.github/workflows/publish.yml

```
@@ -52,21 +35,22 @@ jobs:
               - 'libs/core/**'
       - name: Create npm config
         if: steps.modified-files.outputs.core == 'true'
-        run:  |
+        run: |
           cd libs/core 
           echo '//registry.npmjs.org/:_authToken=${{secrets.NPM_PUBLISH_TOKEN}}' > ~/.npmrc
       - name: Version bump core and update core dependency version in designsystem
         id: bump-core
         if: steps.modified-files.outputs.core == 'true'
-        run: | # For explanation: https://github.com/kirbydesign/designsystem/pull/1722#issuecomment-928181479
+        run:
+          | # For explanation: https://github.com/kirbydesign/designsystem/pull/1722#issuecomment-928181479
```

Formatting seems to be a bit off compared to the old version. Consider reverting, unless it is done intentionally. 

```suggestion
        run: | # For explanation: https://github.com/kirbydesign/designsystem/pull/1722#issuecomment-928181479
```